pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -376,10 +376,13 @@ func TestPostFilter(t *testing.T) {
 				pdbLister: getPDBLister(informerFactory),
 				args:      *getDefaultDefaultPreemptionArgs(),
 			}
+			diagnosis := framework.Diagnosis{
+				NodeToStatusMap: make(framework.NodeToStatusMap),
+			}
 
 			state := framework.NewCycleState()
 			// Ensure <state> is populated.
-			if _, status := f.RunPreFilterPlugins(ctx, state, tt.pod); !status.IsSuccess() {
+			if _, status := f.RunPreFilterPlugins(ctx, state, tt.pod, &diagnosis); !status.IsSuccess() {
```

Comment:
```suggestion
			if _, status := f.RunPreFilterPlugins(ctx, state, tt.pod, &framework.Diagnosis{}); !status.IsSuccess() {
```

We should not need to initialize the `nodeStatus` map.

And same in other tests.